pkg/controller/start.go

```diff
@@ -138,6 +142,112 @@ func createManager(log logr.Logger, cfg config.Manager) (ctrl.Manager, error) {
 		log.Info("Watching all namespaces")
 	}
 	opts.WebhookServer = webhook.NewServer(webhookOpts)
+	ctx, _ := context.WithCancel(context.Background())
+	go func() {
+		log.Info("Starting webhook server in background")
+		if err := opts.WebhookServer.Start(ctx); err != nil && err != http.ErrServerClosed {
```

Comment:
Should be start starting this before we register the handlers? Also, why are we starting it explicitly? Just so we can get the better logging?